### PR TITLE
Add to favourites

### PR DESCRIPTION
**Pull request by ZarechOleksii**

### CODE REVIEW

**Comment by IvanVuzh** [comment]
lgtm

File: src/Models/Entities/Movie.cs [hunk 1]
```diff
@@ -21,5 +21,7 @@ public class Movie : BaseEntity
         [Required]
         [Url]
         public string Url { get; set; }
+
+        public List<Favourites> Favorites { get; set; }
```
Comment (IvanVuzh) [inline src/Models/Entities/Movie.cs:25]: To use "u" or not to use

File: src/Models/Entities/User.cs [hunk 2]
```diff
@@ -9,5 +9,6 @@ namespace Models.Entities
 {
     public class User : IdentityUser
     {
+        public List<Favourites> Favorites { get; set; }
```
Comment (IvanVuzh) [inline src/Models/Entities/User.cs:12]: same here

**Review by IvanVuzh — CHANGES_REQUESTED** [review]

File: src/Models/Entities/Movie.cs (hunk 1 above)
Comment (IvanVuzh) [reply src/Models/Entities/Movie.cs:25]: that is the question

File: src/Models/Entities/User.cs (hunk 2 above)
Comment (IvanVuzh) [reply src/Models/Entities/User.cs:12]: that is the question

File: src/Models/Entities/Movie.cs (hunk 1 above)
Comment (ZarechOleksii) [reply src/Models/Entities/Movie.cs:25]: Boris Johnson > Sleepy Joe
Boris Johnson approves u in Favourite

File: src/Models/Entities/User.cs (hunk 2 above)
Comment (ZarechOleksii) [reply src/Models/Entities/User.cs]: Boris Johnson > Sleepy Joe
Boris Johnson approves u in Favourite

File: src/Models/Entities/Movie.cs (hunk 1 above)
Comment (ZarechOleksii) [reply src/Models/Entities/Movie.cs]: Done

File: src/Models/Entities/User.cs (hunk 2 above)
Comment (ZarechOleksii) [reply src/Models/Entities/User.cs]: Done

File: src/Services/Implementations/FavouriteService.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                MovieId = movieId
+            };
+
+            try
+            {
+                return await _repository.DeleteAsync(favourites);
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex, "Caught exception in Favourite service AddToFavouritesAsync");
```
Comment (ZarechOleksii) [inline src/Services/Implementations/FavouriteService.cs:52]: Change messagee

**Pull request merged**